docs/coding-standards/pr-reviews.md

```diff
@@ -0,0 +1,28 @@
+# Pull Request Review Guidelines
+
+1. Does the PR description follow our template?
+
+2. Does the PR title and description describe the changes (at least at a high level)? (useful for going through commit history later)
+
+3. Does the change satisfy its ticket? (Or have other tickets been created for un-implemented things?)
+
+4. Should this change be made?
+   - does it agree with business requirements?
+   - does it contribute to good architecture?
+   - does it have no security issues? (eg, does not expose passwords)
+
+5. Does the code follow our coding standards?
+   - code style
+   - cleanly written
+   - improves architecture (or at least doesn't worsen it), eg. use of CSS, variables, mixins, etc
+   - adequately commented so reviewers/future devs can understand what it's doing
+   - adequately commented to explain special cases, business logic, etc
+   - has sufficient unit tests to verify all code paths
+   - does not reformat existing code (which, if needed, should be a separate ticket/PR)
```

Comment:
Curious about this point. Maybe i'm unsure of the context but this is something we are doing consistently, house keeping per se.

Reply:
As you know, I am guilty of this :-}

However, the intent is to keep PRs concise and ease reviews.

I am not sure if there is an acceptable middle ground. For example, I suggested a separate ticket or PR, but I think that complicates life for the developer, who likely prefers to clean up code while they're in that area rather than having to remember to do it at a later time.

Let's call this a "best practice" and just be flexible with it if nobody minds.

Reply:
100% on board with that :)